tests/securityhub_same_account/main.tf

```
@@ -15,6 +15,27 @@ module "securityhub" {
   ]
 }
 
+module "standards_control" {
+  source = "../../"
+
+  standards_control = [
+    {
+      standards_control_arn = "arn:aws:securityhub:::ruleset/cis-aws-foundations-benchmark/v/1.2.0"
```

Let's add the name here, and also add a name for the second item in the list:

```suggestion
      name                  = "cis-aws-foundations-benchmark"
      standards_control_arn = "arn:aws:securityhub:::ruleset/cis-aws-foundations-benchmark/v/1.2.0"
```